consumer_test.go

```diff
@@ -23,7 +23,7 @@ func TestConsumer_generateQuery(t *testing.T) {
 		{
 			name: "simple",
 			args: args{queueName: "testing_queue"},
-			want: "UPDATE \"testing_queue\" SET locked_until = $1, started_at = CURRENT_TIMESTAMP, consumed_count = consumed_count+1 WHERE id IN (SELECT id FROM \"testing_queue\" WHERE (locked_until IS NULL OR locked_until < CURRENT_TIMESTAMP) AND consumed_count < 3 AND processed_at IS NULL ORDER BY consumed_count ASC, created_at ASC LIMIT $2 FOR UPDATE SKIP LOCKED) RETURNING id, payload, metadata, consumed_count, locked_until",
+			want: "UPDATE \"testing_queue\" SET locked_until = :locked_until, started_at = CURRENT_TIMESTAMP, consumed_count = consumed_count+1 WHERE id IN (SELECT id FROM \"testing_queue\" WHERE (locked_until IS NULL OR locked_until < CURRENT_TIMESTAMP) AND consumed_count < :max_consume_count AND processed_at IS NULL ORDER BY consumed_count ASC, created_at ASC LIMIT :limit FOR UPDATE SKIP LOCKED) RETURNING id, payload, metadata, consumed_count, locked_until",
```

Comment:
I agree, this is much nicer.

Reply:
There are also some named args in stdlib, Is it necessary to use sqlx?

https://pkg.go.dev/database/sql#Named

Reply:
Support yes, but not implemented by the lib/pq. Leaving this as part of #8 for later